documents/local_initiatives/README.md

```diff
@@ -11,4 +11,83 @@ You need three things to list a group in the directory.
 
 1.  Group location (city or region, if variable)
 2.  Event/group name
-3.  Contact point - Please do not list organizers' personal info here. Either link to a group's web/meetup site or list a public profile only. Twitter or Github are usually best for this. PRs with private info will be asked to be edited
\ No newline at end of file
+3.  Contact point - Please do not list organizers' personal info here. Either link to a group's web/meetup site or list a public profile only. Twitter or Github are usually best for this. PRs with private info will be asked to be edited
+
+## Submitting your initiative
+
+If you want to submit an initiative you're a part of, or know someone who is a part of, you can add it to the list by submitting a PR to the [documents/local_iniatives/current-initiatives.md](./current-initiatives.md) file in this repository.
+
+## Step 1: Add your region.
+
+
+### What should I submit in a PR?
+
+The initiative directory should follow the format below:
+
+```
+# Country OR Major Region
+## More Specific Region
+
+Location | Event | Website | Contact Point
+-------- | ----- | ------- | -------------
+Event Location  | Event Name | <Event Link> | Contact Method Name: Contact Method Address
+```
+
+Where...
+* `Country OR Major Region` is the name of the country or region events below would take place in. Some examples are `North America`, `Europe`, `Australia`, `Asia`, and so on.
```

Comment:
Just a though I had while reading: maybe we can add a place for online initiatives without physical meetings.